lm/perplexity.py

```diff
@@ -33,7 +33,13 @@ def __init__(
 
         self.log_file = self.log_file_output_path("compute_ppl", crp, False)
         self.score_file = self.output_path("word.scores")
-        self.perplexity = self.output_var("perplexity")
+        self.num_tokens = self.output_var("num-tokens")
```

Comment:
Once the Job finished, all output variables should be set. 

In the code below there can be the case, that some of these variables remain unset. I propose either
* add a fallback value here 
* or add else cases below that set some sentinel values

(e.g. 0 or inf, whatever reasonable)